root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -2699,18 +2699,25 @@ const testData = [
        only_valid_entity_types: [],
   },
   // Songkick
+  {
+                     input_url: 'https://www.songkick.com/artists/3909026-courtney-barnett/calendar',
+             input_entity_type: 'artist',
+    expected_relationship_type: 'songkick',
+            expected_clean_url: 'https://www.songkick.com/artists/3909026',
+       only_valid_entity_types: ['artist'],
+  },
   {
                      input_url: 'http://www.songkick.com/festivals/74586-ruhrpott-rodeo/id/19803209-ruhrpott-rodeo-festival-2014',
              input_entity_type: 'event',
     expected_relationship_type: 'songkick',
-            expected_clean_url: 'https://www.songkick.com/festivals/74586-ruhrpott-rodeo/id/19803209-ruhrpott-rodeo-festival-2014',
+            expected_clean_url: 'https://www.songkick.com/festivals/74586/id/19803209',
        only_valid_entity_types: ['event', 'place'],
```

Comment:
Why would festivals be valid for place?

Reply:
After digging the code base for [initial commit](https://github.com/metabrainz/musicbrainz-server/commit/790ffea4522ab5526927806bafd370852c9d4b60#diff-3e811e588fddba5fbac605a42e3e13d9R888) and the database which has no festival linked to place, there seems to be no reason, it looks more like a bug, fixed in https://github.com/metabrainz/musicbrainz-server/pull/1067/commits/e7c9b6529c3d6f77b8923ba7061bd30c741487b4.